app/components/deputy_assignment_component.html.erb

```diff
@@ -1,6 +1,10 @@
 <div id="<%= dom_id deputy_assignment %>" class="<%= root_class %>">
   <div class="deputy-assignment__name">
-    <strong><%= name %></strong> <%= webaccess_id %>
+    <% if current_user_is_deputy? && !pending? %>
```

Comment:
Just wanted to let you know that there is a method for this: `confirmed?`  

`confirmed? == !pending?` (I personally dislike `!`s in conditionals but that's just me)